core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionMonitorActor.scala

```diff
@@ -396,7 +396,7 @@ trait SubmissionMonitor extends FutureSupport with LazyLogging with RawlsInstrum
   }
 
   def updateEntityAndWorkspace(entity: Entity, workspace: Workspace, workflowOutputs: Iterable[BoundOutputExpression]): (WorkflowEntityUpdate, Option[Workspace]) = {
-    val entityUpsert = workflowOutputs.collect({ case BoundOutputExpression(ThisEntityTarget, attrName, attr) => (attrName, attr) })
+    val entityUpsert = workflowOutputs.collect({ case BoundOutputExpression(EntityTarget, attrName, attr) => (attrName, attr) })
```

Comment:
`EntityTarget` can be used by `InputExpression`s to refer to an entity other than `this`